dev-perl/PGObject-Util-DBMethod/PGObject-Util-DBMethod-1.01.000.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DIST_AUTHOR="EHUELS"
+
```

Comment:
DIST_VERSION=1.01.000

and the ebuild is 1.1.0
dev-perl/Gentoo-PerlMod-Version  <-- this gives you the definitive values